core/sr-arithmetic/benches/bench.rs

```diff
@@ -0,0 +1,76 @@
+use criterion::{Criterion, criterion_group, criterion_main};
```

Comment:
License header + how to run

Reply:
You can now just run the benchmarks with `cargo bench` so I'm not sure we need run instructions.

Reply:
okay 👍